package.json

```
@@ -15,6 +15,7 @@
   "scripts": {
     "build": "ember build",
     "lint:js": "eslint .",
+    "lint:blueprints": "tsc -noEmit",
```

Per comment on the `tsconfig.json` file, we should reference the test config instead. Also there's a typo here: should be `--noEmit`.

```suggestion
    "lint:blueprints": "tsc test.tsconfig.json --noEmit",
```